python/src/main/python/drivers/piglit-worker.py

```diff
@@ -48,40 +46,45 @@
 JSON_SUFFIX = '.json'
 PNG_SUFFIX = '.png'
 SHADER_TEST_SUFFIX = '.shader_test'
-PNG_FILENAME = 'shader_runner_gles3000.png'
-LOGFILE_NAME = 'piglit_log.txt'
-STATUS_FILENAME = 'STATUS'
 
 SHADER_RUNNER_ARG_PNG = '-png'
 SHADER_RUNNER_ARG_AUTO = '-auto'
 SHADER_RUNNER_ARG_UNIFORMS = '-ignore-missing-uniforms'
 SHADER_RUNNER_ARG_SUBTESTS = '-report-subtests'
+
 WORKER_INFO_FILE = 'worker_info.json'
+PNG_FILENAME = 'shader_runner_gles3000.png'
+COMPARE_PNG_FILENAME = 'shader_runner_gles3001.png'
+NONDET0_PNG = 'nondet0.png'
+NONDET1_PNG = 'nondet1.png'
+LOGFILE_NAME = 'piglit_log.txt'
+STATUS_FILENAME = 'STATUS'
 
 NO_DRAW_ARG = '--nodraw'
 
-RETURNCODE_STR = 'Returncode: '
-STDOUT_STR = 'STDOUT: '
-STDERR_STR = 'STDERR: '
-
 STATUS_SUCCESS = 'SUCCESS'
 STATUS_CRASH = 'CRASH'
 STATUS_TIMEOUT = 'TIMEOUT'
-STATUS_SANITYERROR = 'SANITY_ERROR'
 STATUS_UNEXPECTED = 'UNEXPECTED_ERROR'
 STATUS_NONDET = 'NONDET'
 
 TIMEOUT = 30
 
+use_catchsegv = True
```

Comment:
Using a global boolean here is kind of hacky, but I think it might be a better option than trying to pass yet another boolean argument all the way down the call stack to run_image_job().